src/main/java/io/aiven/klaw/config/ManageDatabase.java

```diff
@@ -119,14 +133,38 @@ private void shutdownApp() {
     ((ConfigurableApplicationContext) contextApp).close();
   }
 
-  @PostConstruct
-  public void loadDb() throws Exception {
-    handleDbRequests = handleJdbc();
-    loadStaticDataToDb();
-    updateStaticDataToMemory();
-    checkSSOAuthentication();
+  @Override
+  public void afterPropertiesSet() throws KlawException {
+    loadDb();
+  }
+
+  @Override
+  public void destroy() throws Exception {
+    shutdownApp();
+  }
+
+  public void loadDb() throws KlawException {
+    try {
+      loadStaticDataToDb();
+      updateStaticDataToMemory();
+      checkSSOAuthentication();
+    } catch (Exception e) {
+      e.printStackTrace();
+      log.error("Error in starting the application. {}", e.getMessage());
+      throw new KlawException(e.getMessage());
+    }
   }
 
+  public HandleDbRequestsJdbc getHandleDbRequests() {
+    return handleDbRequests;
+  }
+
+  //  @Bean()
+  //  @Conditional(JdbcDataSourceCondition.class)
+  //  HandleDbRequestsJdbc handleJdbc() {
+  //    return new HandleDbRequestsJdbc();
+  //  }
```

Comment:
Do we need this comment or can just simply remove it?

Reply:
We don't need this. deleted.